File: Codeforces/cf1337b.py
# -*- coding: utf-8 -*-
# Leonardo Deliyannis Constantin
# https://codeforces.com/contest/1337/problem/B

def main():
    t = int(input())
    while t > 0:
        t -= 1
        x, n, m = map(int, input().split())
        while x > 20 and n > 0:
            x = x // 2 + 10
            n -= 1
        print("YES" if m * 10 >= x else "NO")

if __name__ == '__main__':
    main()

File: Codeforces/cf1337b.cpp
/*
 * Leonardo Deliyannis Constantin
 * https://codeforces.com/contest/1337/problem/B
*/

#include<bits/stdc++.h>
using namespace std;

int main(){
    int t, x, n, m;
    scanf("%d", &t);
    while(t--){
        scanf("%d %d %d", &x, &n, &m);
        while(x > 20 && n > 0){
            x = (x/2) + 10;
            n--;
        }
        x -= m*10;
        printf("%s\n", (x <= 0) ? "YES" : "NO");
    }
    return 0;
}
